graphics/DDA_Algorithm.c

```diff
@@ -0,0 +1,46 @@
+// DDA line drawing algorithm
```

Comment:
Please describe how the algorithm works, a Wikipedia link, what the algorithm does, etc..